includes/wikia/LBFactory_Wikia.php

```diff
@@ -57,11 +57,8 @@ function getSectionForWiki( $wiki = false ) {
 		}
 		else {
 			// this is a foreign db that either has a cluster defined in WikiFactory...
-			$city_id = WikiFactory::DBtoID( $wiki );
-			$section = WikiFactory::getWikiByID( $city_id )->city_cluster;
-			if ( empty( $section ) ) {
-				$section = 'central';
-			}
+			$wikiInfo = WikiFactory::getWikiByDB( $dbName );
+			$section = $wikiInfo->city_cluster ?? 'central';
```

Comment:
All wikis do have `city_cluster` set. Maybe it's a right time to remove this fallback. Or we can start with logging fallbacks to "central" here.